src/app/auto-complete/auto-complete.component.spec.ts

```diff
@@ -3,6 +3,32 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { AutoCompleteComponent } from './auto-complete.component';
+import {AutocompleteService} from "../autocomplete.service";
```

Comment:
you already have autocomplete service imported and you import for the second time.